src/Moryx.Products.Model/ProductsContext.cs

```diff
@@ -5,15 +5,12 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Moryx.Model;
-using Moryx.Model.Attributes;
-using Moryx.Model.PostgreSQL;
 
 namespace Moryx.Products.Model
 {
     /// <summary>
     /// The DBContext of this database model.
     /// </summary>
-    [ModelConfigurator(typeof(NpgsqlModelConfigurator))]
```

Comment:
We need to do this in all modules right? Then it would be great if you could add it to the migration guide as removed without replacement 😊